introduction.md

```diff
@@ -6,33 +6,31 @@ layout: documentation
+openHAB is **software** for integrating different home automation systems, devices and technologies into one single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+openHAB is also a **community** of users, contributors and maintainers, working together on an open-source, interoperable approach to home automation.
 
 ## openHAB Structure
 
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+openHAB 2 is based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework, and is written in Java.
+It uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) to create an Open Services Gateway initiative (OSGi) runtime environment.
+[Jetty](https://www.eclipse.org/jetty/) is used as an HTTP server.
 
-openHAB is highly modular software, which means that the base installation (the "runtime") can be extended through different kinds of "add-ons", either to communicate with new home automation solution, or to offer a new kind of user interface.
+openHAB is highly modular software, and can be extended through "add-ons".
+Add-ons give openHAB a wide array of capabilities, from User Interfaces, to the ability to interact with a large and growing number of physical Things.
 
-The structure of openHAB can be summarized this way:
+The overall architecture of openHAB is shown in the figure below:
 
 ![distribution overview](images/distro.png)
 
-Add-ons can come from three different origins:
-
-* Add-ons coming from the Eclipse SmartHome project, as Eclipse SmartHome is the base of openHAB 2,
-* Add-ons coming from the openHAB version 1 package, as openHAB 2 includes a compatibility layer,
-* Add-ons made for openHAB 2.
-
-
 ## The openHAB Community
 
 openHAB is not only a solution but also a community.
 
 The center of this community is the [openHAB community forum](https://community.openhab.org).
 
-If you have questions about the use of openHAB, before reporting an issue, or for comments and contributions about the documentation, please check the [openHAB community forum](https://community.openhab.org) for solutions and help. More people will see your issue when it is posted in the forum and community members may be able to help you find known solutions/workarounds for your issue.
+If you have questions about the use of openHAB, before reporting an issue, please check the [openHAB community forum](https://community.openhab.org) for solutions and help.
```

Comment:
My suggestion would be:

>If you have questions about the use of openHAB, please check the openHAB community forum for solutions and help  before reporting an issue.

Reply:
Yes - I agree and will make that change.

Reply:
I wonder if we could even strip the issue topic. A normal user shouldn't even think about that.

Reply:
I agree with @ThomDietrich.  Let's pull this out entirely.  I will make this change.

Reply:
This issue has been resolved.